sass/styles.scss

```diff
@@ -1,3 +1,4 @@
+@import "github";
```

Comment:
Nitpic: an empty line below this one would be nice.

Reply:
added